addons/mass_mailing_sale/data/mass_mailing_demo.xml

```diff
@@ -6,6 +6,7 @@
         <field name="subject">Our last promotions, just for you !</field>
         <field name="state">in_queue</field>
         <field name="user_id" ref="base.user_admin"/>
+        <field name="email_from" eval="base.user_admin.email_formatted"/>
```

Comment:
@tde-banana-odoo Here I had to set the email_from because during the installation of the demo data, the compute was not called...
See: https://runbot.odoo.com/runbot/build/8236637